src/client/components/pages/privacy.js

```diff
@@ -20,6 +20,10 @@ const React = require('react');
 
 const PageHeader = require('react-bootstrap').PageHeader;
 
+/**
+* Renders the document and displays the 'privacy policy of BookBrainz'
+* @returns {ReactElement} a HTML file is displayes the privacy policy
```

Comment:
I'd change this to:

    @returns {ReactElement} an element containing the rendered output

The element is combined with others to form the complete HTML file which gets sent to the client, so at this point there isn't an HTML file.